src/lib/comms/sol-socket-dtls-impl-tinydtls.c

```diff
@@ -655,6 +656,98 @@ get_psk_info(struct dtls_context_t *ctx, const session_t *session,
     return r;
 }
 
+static int
+get_ecdsa_key(struct dtls_context_t *ctx, const session_t *session,
+    const dtls_ecdsa_key_t **result)
+{
+    struct sol_socket_dtls *socket = dtls_get_app_data(ctx);
+    struct sol_network_link_addr addr;
+    static unsigned char ecdsa_priv_key[SOL_DTLS_ECDSA_PRIV_KEY_LEN],
+        ecdsa_pub_key_x[SOL_DTLS_ECDSA_PUB_KEY_X_LEN],
+        ecdsa_pub_key_y[SOL_DTLS_ECDSA_PUB_KEY_Y_LEN];
+    static dtls_ecdsa_key_t ecdsa_key;
```

Comment:
Ditto. (actually only this one would be present inside the `sol_socket_dtls` and the auxiliary vars above would not be needed.)